### PR TITLE
Add sciserver information to README

### PR DESCRIPTION
**Pull request by zoghbi-a**
Address #133

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,16 @@ This badge opens Jupyterlab session on Binder which can be used to run the works
 being left unattended for several minutes.
 
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/NASA-NAVO/notebooks/main?urlpath=lab)
+
+
+## Running on Sciserver
+
+The workshop notebooks can also be run on [Sciserver](https://sciserver.org/), which offer an online jypyter platform. To use it for the workshop, follow these steps:
```
Comment (bsipocz) [inline README.md:30]:
```suggestion
The workshop notebooks can also be run on [Sciserver](https://sciserver.org/), which offers an online Jupyter platform. To use it for the workshop, follow these steps:
```

File: README.md [hunk 2]
```diff
@@ -23,3 +23,16 @@ This badge opens Jupyterlab session on Binder which can be used to run the works
...
+
+- Create an account on sciserver, if you do not already have one.
```
Comment (bsipocz) [inline README.md:32]:
```suggestion
- Create an account on Sciserver, if you do not already have one.
```

File: README.md [hunk 3]
```diff
@@ -23,3 +23,16 @@ This badge opens Jupyterlab session on Binder which can be used to run the works
...
+
+- Once logged in, select the *Compute* app.
+
+- Create a container, give it name, select the *'Sciserver Essentials 2.0'* image, and click *Create*. Click on the container name to launch the jupyterlab interface.
```
Comment (bsipocz) [inline README.md:36]:
```suggestion
- Create a container, give it name, select the *'Sciserver Essentials 2.0'* image, and click *Create*. Click on the container name to launch the JupyterLab interface.
```

**Review by bsipocz — APPROVED** [review]
super minor fixes. 

CI failures are present on `main`, so should not be a blocker for this one.

**Comment by zoghbi-a** [comment]
Sciserver now has a separate compute image called: 'NAVO-workshop'. The text in the README file has been updated to show how to use it.

**Pull request merged**